kokoro/config/build/presubmit/bullseye_x86_64.gcl

```diff
@@ -3,7 +3,7 @@ import '../common.gcl' as common
 config build = common.build {
   params {
     environment {
-      DISTRO = 'bullseye'
```

Comment:
revert these temporary changes and everything else looks good!

Reply:
Done